Assets/Scripts/SS3D/Systems/Gamemodes/UI/GamemodeObjectivePanelView.cs

```diff
@@ -19,9 +18,26 @@ public class GamemodeObjectivePanelView : Actor
 
         [SerializeField] private GamemodeObjectiveItemView _itemViewPrefab;
         [SerializeField] private GameObject _content;
+        private Controls.OtherActions _controls;
 
         private Dictionary<int, GamemodeObjectiveItemView> _gamemodeObjectiveItems;
 
+        private void OnEnable()
+        {
+            try
+            {
```

Comment:
replace with` if(null && !didstart) return; else { Debug.Error(); return; }` So we can still have an error in case performed is null later in the game.